dataactcore/migrations/versions/c5449ab3577f_add_element_numbers_column_to_file_.py

```diff
@@ -0,0 +1,42 @@
+"""Add element_numbers column to file_generation table
+
+Revision ID: c5449ab3577f
+Revises: 6e3be68b87ae
+Create Date: 2020-08-06 15:38:13.802389
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'c5449ab3577f'
+down_revision = '0d570ef8292d'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+
+
+
+def upgrade_data_broker():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('file_generation', sa.Column('element_numbers', sa.Boolean(), nullable=True))
+    op.execute("UPDATE file_generation SET element_numbers = false")
+    op.alter_column('file_generation', 'element_numbers', nullable=False)
+    # ### end Alembic commands ###
+
+
+def downgrade_data_broker():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('file_generation', 'element_numbers')
+    # ### end Alembic commands ###
```

Comment:
I'm kind of confused why it's done like this. Couldn't the `sever_default` be set to `false` and just add the column with `nullable` as false?

Reply:
I literally did that and it didn't work... don't know why it decided to play nice now that you requested it. Maybe I forgot the quotes around it or something last time?